core/src/js/models/duels/duels-types.ts

```diff
@@ -1,6 +1 @@
-export type DuelsTopDecksDustFilterType = 'all' | '1000' | '500' | '200' | '100' | '40' | '0';
-export type DuelsUnlocksFilterType =
```

Comment:
Why did you remove that type declaration? I assume it's a mistake?

Reply:
Now all the treasures are open by default and you removed corresponding filter from app and I thought that we can also remove this declaration. This type is currently not used anywhere in the app.